### PR TITLE
Major release

### PR DESCRIPTION
**Pull request by alessioventuriniAND**
### What is the context of this PR?

This Pr gets all our breaking changes ready for our major release on the 9th of October. 

The below PRs have been merged in the major-release branch : 

- #3111 
- #3188
- #3268 
- #3275 
- #3143 
- #3290 
- #3291 
- #3227 
- #3294 
- #3293 
- #3292
- #3295 
- #3224 
- #3180 
- #3299 
- #3300 
- #3303 
- #3304 
- #3305 
- #3306 
- #3307
- #3336 
- #3382
- #3380 
- #3379 
- #3377 
- #3285 

### How to review this PR

Double check that all the PR that have been merged are correctly working as expected.

### Checklist

This needs to be completed by the person raising the PR.

-   [x] I have selected the correct Assignee
-   [x] I have linked the correct Issue

### CODE REVIEW

File: migration_guides/70.x.x-to-71.0.0-migration-guide.md [hunk 1]
```diff
@@ -0,0 +1,1042 @@
...
+        {{
+            onsMessage({
+                "unreadLink": "www.google.com"
+            })
+        }}
+
+        NEW
+        {{
+            onsMessage({
+                "unreadLinkUrl": "www.gooogle.com"
```
Comment (rmccar) [inline migration_guides/70.x.x-to-71.0.0-migration-guide.md:336]:
```suggestion
                "unreadLinkUrl": "www.google.com"
```

File: migration_guides/70.x.x-to-71.0.0-migration-guide.md [hunk 2]
```diff
@@ -0,0 +1,1042 @@
...
+        {{
+            onsImage({
+                "url": "www.google.com"
+            })
+        }}
+
+        NEW
+        {{
+            onsImage({
+                "src": "www.gooogle.com"
```
Comment (rmccar) [inline migration_guides/70.x.x-to-71.0.0-migration-guide.md:484]:
```suggestion
                "src": "www.google.com"
```

File: migration_guides/70.x.x-to-71.0.0-migration-guide.md [hunk 3]
```diff
@@ -0,0 +1,1042 @@
...
+           OLD
+           {{
+               onsDocumentList({
+                   "url": '#0',
+                   "title": 'ONS launches Integrated Data Service to boost government collaboration on data sharing',
+                   "metadata": {
+                       "type": {
+                           "text": 'Press releases',
+                           "ref": 'Ref 008052'
+                       },
```
Comment (rmccar) [inline migration_guides/70.x.x-to-71.0.0-migration-guide.md:744]:
```suggestion
                   "metadata": {
                       "type": {
                           "text": 'Press releases',
                           "ref": 'Ref 008052'
                       },
                   },
```

File: migration_guides/70.x.x-to-71.0.0-migration-guide.md [hunk 4]
```diff
@@ -0,0 +1,1042 @@
...
+               onsDocumentList({
+                   "title": {
+                       "text": 'ONS launches Integrated Data Service to boost government collaboration on data sharing',
+                       "url": '#0'
+                   },
+                   "metadata": {
+                       "object": {
+                           "text": 'Press releases',
+                           "ref": 'Ref 008052'
+                       },
```
Comment (rmccar) [inline migration_guides/70.x.x-to-71.0.0-migration-guide.md:759]:
```suggestion
                   "metadata": {
                       "object": {
                           "text": 'Press releases',
                           "ref": 'Ref 008052'
                       },
                   },
```

**Review by rmccar — CHANGES_REQUESTED** [review]
Just noticed a couple of typos

File: src/components/address-input/_macro-options.md [hunk 5]
```diff
@@ -28,8 +29,8 @@
...
-| errorMessageAPILinkText | string                                                        | false                            | Link text used to toggle to a manual mode when using the address macro                                                                             |
+| errorMessageApi         | string                                                        | false                            | Error message displayed when the API has failed during a search                                                                                    |
+| errorMessageApiLinkText | string                                                        | false                            | Link text used to toggle to a manual mode when using the address macro                                                                             |
 | manualLink              | string                                                        | false                            | url for the link displayed below the input                                                                                                         |
```
Comment (precious-onyenaucheya-ons) [inline src/components/address-input/_macro-options.md:34]: The manualLink has been updated to manualLinkUrl

**Review by rmccar — APPROVED** [review]

**Review by precious-onyenaucheya-ons — APPROVED** [review]

**Pull request merged**